ui/src/components/flows/ValidationError.vue

```diff
@@ -1,10 +1,20 @@
 <template>
-    <el-tooltip :disabled="!error" raw-content transition="" :hide-after="0" :persistent="false">
+    <el-tooltip
+        :disabled="!error"
+        raw-content
+        transition=""
+        :hide-after="0"
+        :persistent="false"
+        :effect="theme === 'light' ? 'light' : none"
```

Comment:
Tooltip are always dark even on light theme. 
Please just add some style for the text

Reply:
is white font ok for both @tchiotludo  ? 
![image](https://github.com/kestra-io/kestra/assets/37600690/234068e3-9870-4dfb-9aa8-0bffec3eff4b)
![image](https://github.com/kestra-io/kestra/assets/37600690/3b886e0a-a8f6-4ea6-aaf3-33092c04b1bf)

Reply:
prefect :+1: